sdk/communication/azure-communication-identity/src/samples/java/com/azure/communication/identity/CreateAndIssueToken.java

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.communication.identity;
+
+import com.azure.communication.common.CommunicationUserIdentifier;
+import com.azure.communication.identity.models.CommunicationTokenScope;
+import com.azure.core.credential.AccessToken;
+import com.azure.core.credential.AzureKeyCredential;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class CreateAndIssueToken {
+    public static void main(String[] args) {
+        String endpoint = System.getenv("AZURE_COMMUNICATION_ENDPOINT");
```

Comment:
With samples, we usually put a fake value. E.g.
String endpoint = "https://<RESOURCE_NAME>.communication.azure.com"; String accessKey = "SECRET";

Reply:
to be in sync with other sdks we what to make this a runable example

Reply:
See comment above